src/components/Home/Home.js

```diff
@@ -109,19 +113,44 @@ class Home extends Component {
 							user={user}
 						/>
 					</div>
+					<div className="HoverProfilePlaceholder" />
 				</div>
 			);
 
 		return (
 			<div className="Home">
-				<Header>
-					<p className="logout" onClick={this.logout.bind(this)}>Logout</p>
-					<p className="logout" onClick={this.account.bind(this)}>Account</p>
-				</Header>
+				<Header 
+					isLoggedIn
+					userId = {user ? user._id : null}
+					activePage="home"
+					settingsClick= {this.account.bind(this)}
+					/>
 				{content}
 			</div>
 		);
 	}
 }
 
 export default withRouter(Home);
+
+
+/*<div className="HeaderButton">
+			        <img className="HeaderChannel" src={channelIcon}/>
+			        <div>Channels</div>
+			      </div>
+			      {/* Re-enable once Notifications page is created
+			      <div className="HeaderButton" onClick={this.notifications.bind(this)}>
+			        <img className="HeaderNotif" src={notifIcon}/>
+			        <div>Notifications</div>
+			      </div>
+			  
+					<div className="HeaderButton" >
```

Comment:
Minor detail, but when you hover over the profile picture, it looks like it should be clickable but nothing actually happens when you click it.